client/notifications/index.jsx

```diff
@@ -283,21 +304,30 @@ export class Notifications extends Component {
 		}
 
 		return (
-			<div
-				id="wpnc-panel"
-				className={ clsx( 'wide', 'wpnc__main', {
-					'wpnt-open': this.props.isShowing,
-					'wpnt-closed': ! this.props.isShowing,
-				} ) }
-			>
-				<NotificationsPanel
-					actionHandlers={ this.actionHandlers }
-					isShowing={ this.props.isShowing }
-					isVisible={ this.state.isVisible }
-					locale={ localeSlug }
-					wpcom={ wpcom }
-				/>
-			</div>
+			<>
+				{ /*
+					Due to how the notifs panel width is set differently on this page to control fly-out vs. in place
+					nested levels, this notice makes sense at different places in the DOM depending on the
+					breakpoint. We remove display for one or the other via CSS depending on the breakpoint.
+				*/ }
+				<Notifications3PCNotice className="reader-notifications__3pc-notice-external" />
```

Comment:
Could we hide the notice if you have 3PC? Or it's not worth the check?

Reply:
I don't think its worth the check. I think our plans might be to even remove this from sidebar navigation so that users can only get here if they click their notifications bell outside of calypso when having the 3PCs disabled.